src/Auth0.OidcClient.Core/Tokens/IdTokenRequirements.cs

```diff
@@ -36,6 +37,10 @@ internal class IdTokenRequirements
         /// <summary>
         /// Required organization the token must be for.
         /// </summary>
+        /// <remarks>
+        /// - If you provide an Organization ID (a string with the prefix `org_`), it will be validated against the `org_id` claim of your user's ID Token. The validation is case-sensitive.
+        /// - If you provide an Organization Name (a string *without* the prefix `org_`), it will be validated against the `org_name` claim of your user's ID Token.The validation is case-insensitive.
```

Comment:
```suggestion
        /// - If you provide an Organization Name (a string *without* the prefix `org_`), it will be validated against the `org_name` claim of your user's ID Token. The validation is case-insensitive.
```